tools/gendocs.sh

```diff
@@ -27,3 +27,10 @@ echo "===================================== Making HTML..."
 makeinfo --html --no-split ledger3.texi
 echo "===================================== Making PDF..."
 texi2pdf --quiet --batch ${papersize} ledger3.texi
+
+echo "===================================== Making Info..."
+makeinfo ledger-mode.texi
+echo "===================================== Making HTML..."
+makeinfo --html --no-split ledger-mode.texi
+echo "===================================== Making PDF..."
+texi2pdf --quiet --batch ${papersize} ledger-mode.texi
```

Comment:
How about a small shell for loop to remove the duplicate code?

        for info in ledger3.texi ledger-mode.texi; do
          echo "===================================== Making Info..."
          makeinfo ${info}
          echo "===================================== Making HTML..."
          makeinfo --html --no-split ${info}
          echo "===================================== Making PDF..."
          texi2pdf --quiet --batch ${papersize} ${info}
        done